Feature/2025/preferences

b25a0031 opened the pull request:
router.js
vue.config.jsの編集
各vueファイルの作成

b25a0031 committed:
vueファイル作成

b25a0031 committed:
router.js/vue.config.jsの編集

b25a0031 committed:
usersDeleteのスぺルミス修正

NishidaR reviewed (commented):
フォルダ名ですが、特別な理由がなければ大文字始まりか小文字始まりかは統一してください。（小文字がおすすめです。言語や環境によるのかもしれないですが、パッケージ名に大文字を使用しているのを見たことがないのが理由です。）

NishidaR commented on front/vue.config.js:
```diff
@@ -12,23 +12,131 @@ module.exports = {
+      title: "ログイン",
     },
-    signOut: {
+    logOut: {
```
ログアウトって必要なんでしたっけ・・？
もし不要であれば削除してください。必要であれば返信お願いします。
※不要の場合はrouter側も不要です。

NishidaR commented on front/vue.config.js:
```diff
@@ -12,23 +12,131 @@ module.exports = {
+      filename: "pages/menu.html",
+      title: "メニュー",
+    },
+    header: {
```
これいらないと思います。
ヘッダーのみを表示したいことはないはずなので。

b25a0031 committed:
レビュー反映
*ファイル名を小文字始まりに統一
*LogOut.vueの削除
*components/Menu.vueの削除
*router.jsおよびvue.config.jsからLogOut.vueの記述を削除
*vue.config.jsからheader.vueの記述を削除

b25a0031 committed:
レビュー反映ミス修正
*router.jsの変更
